sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -20,14 +20,16 @@
     AzureFunctionEnvironmentVariable,
     AzureWebAppEnvironmentVariable,
     ContainerAppEnvironmentVariable,
-    KubernetesEnvironmentVariable
+    KubernetesEnvironmentVariable,
+    EMPTY_LABEL
 )
 
 from ._user_agent import USER_AGENT
 
 if TYPE_CHECKING:
     from azure.core.credentials import TokenCredential
 
+JSON = Mapping[str, Any]  # pylint: disable=unsubscriptable-object
```

Comment:
Nitpick: I would create a type alias for the `Union[str, Mapping[str, Any]]` instead (since we are only using JSON as a type parameter for exactly that)